robot/pmm/tests/browser/ProgramEngagement/ProgramEngagement.robot

```diff
@@ -18,59 +19,96 @@ Setup Test Data
     &{contact} =                   API Create Contact
     Set suite variable             &{contact}
     &{program} =                   API Create Program
-    Store Session Record           ${ns}Program__c         &{program}[Id]
```

Comment:
Did you remove this because API keywords automatically store the session records ?

Reply:
yes, thats correct!